SmartDeviceLink/SDLSyncMsgVersion.h

```diff
@@ -24,6 +24,8 @@
 - (instancetype)initWithDictionary:(NSMutableDictionary *)dict;
 
 // TODO: (Alex M.)[2016-12-1] Change from NSInteger to UInt8
+- (instancetype)initWithMajorVersion:(NSInteger)majorVersion minorVersion:(NSInteger)minorVersion __deprecated_msg("Include patchVersion");
```

Comment:
The deprecated message should match other deprecated messages in the project and tell which method is the recommended one to call.